build_daemon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.9-dev
```

Comment:
We should be able to publish this right away I think so we can drop the `-dev` (in the pubspec also).